website/app/styles/pages/application/hero.scss

```
@@ -3,8 +3,23 @@
 .doc-page-hero {
   display: flex;
   align-items: center;
-  justify-content: center;
-  padding: 120px;
+  justify-content: flex-start;
+  padding: 210px 120px;
```

```suggestion
  padding: 160px 120px;
```

Now that I've seen it in the browser, I think it'd be a good idea to try to surface the cards a bit higher on the page, so looking to reduce the padding in the hero.